qa/rpc-tests/feature_fedpeg.py

```diff
@@ -140,13 +153,21 @@ def test_pegout(self, parent_chain_addr, sidechain):
                 break
         assert pegout_tested
 
+    # TODO Remove when signet supports generate
```

Comment:
Is there an upstream issue to track for this?

Reply:
No, all we have for now is https://github.com/bitcoin/bitcoin/compare/master...kallewoof:signet where I'm commenting directly on the code.
Pinging @kallewoof since I can't remember if I suggested this to him.

Reply:
I intend to make a PR against bitcoin core. For now there is a PR against my repo here: https://github.com/kallewoof/bitcoin/pull/4